src/librustc_target/spec/x86_64_harvey.rs

```diff
@@ -0,0 +1,32 @@
+// This defines the amd64 target for the Linux Kernel. See the linux-kernel-base module for
```

Comment:
This comment needs some updating :laughing:

Reply:
oops :-)